sys/shell/commands/sc_gnrc_udp.c

```diff
@@ -158,22 +160,19 @@ int _gnrc_udp_cmd(int argc, char **argv)
 
     if (strcmp(argv[1], "send") == 0) {
         uint32_t num = 1;
-        uint32_t delay = 1000000;
+        uint32_t delay;
         if (argc < 5) {
-#if IS_USED(MODULE_ZTIMER_MSEC)
-            printf("usage: %s send "
-                   "<addr> <port> <data> [<num> [<delay in ms>]]\n", argv[0]);
-#else
             printf("usage: %s send "
                    "<addr> <port> <data> [<num> [<delay in us>]]\n", argv[0]);
```

Comment:
```suggestion
#if IS_USED(MODULE_ZTIMER_USEC) || IS_USED(MODULE_XTIMER)
            printf("usage: %s send "
                   "<addr> <port> <data> [<num> [<delay in us>]]\n", argv[0]);
#elif IS_USED(MODULE_ZTIMER_MSEC)
            printf("usage: %s send "
                   "<addr> <port> <data> [<num> [<delay in ms>]]\n", argv[0]);
#endif
```

Reply:
I'd say the user interface should always be in µs, otherwise this is just too confusing for the CLI operator.